packages/core/src/App/Containers/NotificationsDialog/notifications-list.tsx

```diff
@@ -40,6 +41,14 @@ const NotificationsList = observer(() => {
         }
     };
 
+    const onActionTrackEvent = (key: string) => {
+        Analytics.trackEvent('ce_notification_form', {
+            action: 'click_cta',
+            form_name: 'ce_notification_form',
+            notification_key: key,
+        });
+    };
```

Comment:
are we tracking all notifications with this card?

Reply:
@amina-deriv, yes, this the requirement for the card